statsmodels/gam/smooth_basis.py

```diff
@@ -0,0 +1,821 @@
+# -*- coding: utf-8 -*-
+"""
+Spline and other smoother classes for Generalized Additive Models
+
+Author: Luca Puggini
+Author: Josef Perktold
+
+Created on Fri Jun  5 16:32:00 2015
+"""
+
+from __future__ import division
+# import useful only for development
+from abc import ABCMeta, abstractmethod
+from statsmodels.compat.python import with_metaclass
+
+import numpy as np
+import pandas as pd
+from patsy import dmatrix
+from patsy.mgcv_cubic_splines import _get_all_sorted_knots
+
+from statsmodels.tools.linalg import matrix_sqrt, transf_constraints
+
+
+# Obtain b splines from patsy
```

Comment:
does this comment refer to _equally_spaced_knots?

Reply:
if so, put it in docstring?  or at least immediately-adjacent to the relevant function?